backend/server.js

```diff
@@ -45,6 +43,4 @@ app.get("/monthlyworkinghours", (req, res) => {
   );
 });
 
-app.listen(port, () => {
-  console.log(`Listening on port ${port}.`);
```

Comment:
Actually this console log is useful so you can easily see on which port the server is running